src/tiled/documentmanager.cpp

```diff
@@ -525,6 +530,42 @@ void DocumentManager::closeCurrentDocument()
     closeDocumentAt(index);
 }
 
+void DocumentManager::closeOtherDocuments(int index)
+{
+    if (index == -1)
+        return;
+
+    mMultiDocumentClose = true;
+
+    for (int i = mTabBar->count(); i > 0; --i) {
+        if (i-1 != index)
+            documentCloseRequested(i-1);
+
+        if (!mMultiDocumentClose)
+            return;
+    }
+
+    mMultiDocumentClose = false;
```

Comment:
This line is superfluous since the value doesn't matter after this. Same in `closeDocumentsToRight`.